docs/guides/how_to_guides/creating_and_editing_expectations/how_to_contribute_a_new_expectation_to_the_shared_library.rst

```diff
@@ -3,40 +3,55 @@
+    There are four Expectation classes that make the development of particular types of Expectations significantly easier by hiding the "crud" code and letting you focus on the business logic of your Expectation. Decide which one suites your Expectation:
 
-        - ``ColumnMapExpectations`` are evaluated for a single column, producing True or False on a row by row basis.
-        - ``ColumnAggregateExpectation`` are also evaluated for a single column, but produce an aggregate metric, such as a mean, standard deviation, number of unique values, type, etc.
-        - ``ColumnPairMapExpectation`` are similar to ``ColumnMapExpectations``, except that they are based on two columns, instead of one.
+        - ``ColumnMapExpectation`` - Expectations of this type validate a single column of tabular data. First they ask a yes/no question from every row in that column. Then they ask what percentage of rows gave a positive answer to the first question. If the answer to the second question is above a specified threshold, the Expectation considers the data valid.
+        - ``ColumnExpectation`` s are also evaluated for a single column, but produce an aggregate metric, such as a mean, standard deviation, number of unique values, type, etc.
+        - ``ColumnPairMapExpectation`` s are similar to ``ColumnMapExpectations``, except that they are based on two columns, instead of one.
         - ``TableExpectation`` are a generic catchall for other types of Expectations applied to tabular data.
 
-    For more details on classes of Expectations, please check out {{link to doc in Core Concepts.}}
 
     Find the appropriate template file in ``great_expectations/examples/expectations/``.
 
-#. Pick a name for your Expectation and copy the template file into the appropriate `contrib/` directory.
+#. Copy the template file into the appropriate `contrib/` directory.
 
-    {{Explain naming conventions for Expectations.}}
+    Recently we introduced a fast-track release process for Expectations that a contributor places in one of the subdirectories of ``contrib``.
+    They are released as PyPI packages separate from ``great-expectations``. When you create a new Expectation in ``contrib/experimental/great_expectations_experimental/expectations/``,
+    once your PR is approved and merged, a new version of PyPI package ``great-expectations-experimental``is automatically published.
 
-    {{Explain the roles of `contrib/` and `experimental_expectations/`}}
+#. Pick a name for your Expectation, rename the file and the class within it.
 
-    If your Expectation was called ``ExpectColumnValuesToEqualThree``, you would copy it to ``contrib/experimental_expectations/expectations/expect_column_values_to_equal_three.py``
+    Great Expectations follows a naming convention. Classes that implement Expectations have CamelCase names (e.g., "ExpectColumnValuesToBeThree"). The framework will
```

Comment:
There's already a style guide for this. The description you have here looks great. Can you add a link to the style guide for those who want to go deeper?

Reply:
Absolutely - a great suggestion!